package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bond-sdk-cards",
-  "version": "1.4.1",
+  "version": "1.5.1",
```

Comment:
Would this be 1.5.0 or did we already have 1.5.0 and now there's a bugfix? The way we're using this is: breaking_change.new_feature.bugfix